### ISSUE TITLE
Test failure of EB 2.6.0

### ISSUE THREAD

**Issue by jhein32** [hpcugent/easybuild-framework#1575]
Hi,

First of all thanks for the new Easybuild 2.6.0.  I installed and tested it and get two errors I haven't seen in earlier versions.  Could you please comment?  Screenshoot attached:

![screen shot 2016-01-26 at 18 27 13](https://cloud.githubusercontent.com/assets/15908212/12588775/8afb3322-c45a-11e5-9c79-80babad4f985.png)

Best wishes
  Joachim

**Comment by boegel** [comment]
Thank for reporting this.

First of all, it's nothing to worry about, since this only concerns stand-alone scripts that are packaged along with EasyBuild. `eb` and everything behind it is fine. `eb` doesn't ever call out to these scripts, and you probably don't need them yourself.

It would be interesting to figure out why these tests are failing, however...

How did you install EasyBuild? If I would guess, I'd say this is a bootstrapped installation (i.e. you're using the latest version of vsc-base underneath)?

What do you get if you run this (with the `EasyBuild/2.6.0` module loaded, or your `$PYTHONPATH` set up to use EB v2.6.0)?

```
python -c "import vsc; print vsc.__file__; import vsc.utils; print vsc.utils.__file__"
```

**Comment by boegel** [comment]
@jhein32: ping on the questions above?

**Comment by jhein32** [comment]
Hi,

Still on my todo list, but we deployed a new cluster (software mostly deployed by EasyBuild)  and I am overwhelmed with user queries, so that took a back seat.

I may have another look towards the end of next week.  Sorry for being slow on this.

Best wishes
  Joachim


On 09 Mar 2016, at 16:25, Kenneth Hoste <notifications@github.com<mailto:notifications@github.com>> wrote:


@jhein32<https://github.com/jhein32>: ping on the questions above?

**Comment by jhein32** [comment]
Hi Kenneth,

I just installed the new EB 2.7.0 and did not get these anymore.  So unless you tell me differently (and are keen on the test), I suggest to close the issue.

I got some new issues, but will file a separate entity for this.

Best wishes
  Joachim

On 09 Mar 2016, at 16:31, Joachim Hein <Joachim.Hein@math.lu.se<mailto:Joachim.Hein@math.lu.se>> wrote:

Hi,

Still on my todo list, but we deployed a new cluster (software mostly deployed by EasyBuild)  and I am overwhelmed with user queries, so that took a back seat.

I may have another look towards the end of next week.  Sorry for being slow on this.

Best wishes
  Joachim


On 09 Mar 2016, at 16:25, Kenneth Hoste <notifications@github.com<mailto:notifications@github.com>> wrote:


@jhein32<https://github.com/jhein32>: ping on the questions above?
